docs/architecture/adr-042-blockchain-riri-org.md

```diff
@@ -0,0 +1,292 @@
+# ADR 042: Blockhchain Reactor Riri-Org
```

Comment:
ADR numbers should be registered in #2313 before being used. Just edit the comment there to add a new one.